### DISCUSSION TITLE
The script output by the gdsfactory.generic_tech.get_klayout_pyxs function does not work properly.

### DISCUSSION THREAD

**Discussion by LuYouxi** [gdsfactory/gdsfactory#2842] [Q&A]
When I output the script using gdsfactory.generic_tech.get_klayout_pyxs, I find that it does not work properly in klayout. The high probability of initial debug discovery is related to the deposit() function. The error message for klayout is as follows:
![image](https://github.com/gdsfactory/gdsfactory/assets/35725059/16b8d984-514c-4b69-89c1-54837964712d)
How to solve this problem?

**Comment by bjokisch** [comment]
I have also reproduced this error. Did you manage to fix it @LuYouxi?
